website/docs/r/cognitive_account_ai_services.html.markdown

```diff
@@ -0,0 +1,150 @@
+---
+subcategory: "Cognitive Services"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cognitive_account_ai_services"
+description: |-
+  Manages a Cognitive AI Services Account.
```

Comment:
Please note "Cognitive Services" was rebranded to Azure AI services" a year ago. 
```suggestion
  Manages an Azure AI Services Account.
```